src/main/java/com/faforever/client/chat/MatchmakingChatController.java

```diff
@@ -68,6 +79,15 @@ public void setChannel(String partyName) {
     setReceiver(partyName);
     matchmakingChatTabRoot.setId(partyName);
     matchmakingChatTabRoot.setText(partyName);
+    String topic = i18n.get("teammatchmaking.chat.topic");
+    topicText.getChildren().clear();
+    Arrays.stream(topic.split("\\s"))
+        .forEach(word -> {
+          Label label = new Label(word + " ");
+          label.setStyle("-fx-font-weight: bold; -fx-font-size: 1.1em;");
+            topicText.getChildren().add(label);
+        });
+    topicText.getChildren().add(discordLink);
```

Comment:
If this is always the same it doesnt need to be reset with every setChannel. I would either move it out of the setChannel or make it a true topic for the channel.

Reply:
setChannel only runs once per controller, because currently we clear the tab and make a new one whenever the party owner changes.
setChannel is basically the initialize function.